lib/cdo/hamburger.rb

```diff
@@ -297,8 +297,12 @@ def self.get_header_contents(options)
     ]
 
     non_en_signed_out_links = [
```

Comment:
Wonder: Can we instead remove the special casing of english vs non-english so these stay consistent?

Reply:
Yes! Ill make sure to do this for the others as well

Reply:
Yes! Ill make sure to do this from here on out